tests/tests_fabric/test_fabric.py

```diff
@@ -514,7 +514,7 @@ def test_setup_dataloaders_replace_standard_sampler(shuffle, strategy):
         ("cpu", "cpu"),
         pytest.param("cuda", "cuda:0", marks=RunIf(min_cuda_gpus=1)),
         pytest.param("gpu", "cuda:0", marks=RunIf(min_cuda_gpus=1)),
-        pytest.param("tpu", "xla:0", marks=RunIf(tpu=True, standalone=True)),
+        pytest.param("tpu", "xla:1", marks=RunIf(tpu=True, standalone=True)),
```

Comment:
The default index in XLA is 1. See https://github.com/Lightning-AI/lightning/pull/17227/files#diff-b4a73d07c0b11aabd5fc15d6d1ff242d19049b45acd10baa3998a3682bcd2f29R53-R54. This was a bug in Fabric

Reply:
I remember changing this to 0 because I was convinced it was a bug. Definitely odd that it is 1 indexed

Reply:
It is 0 indexed but index 0 maps to the CPU